lib/external_service_new/hubspot_downloader.rb

```diff
@@ -1,5 +1,3 @@
-require "seevibes/external_service/hubspot_dispatcher"
-
 module ExternalServiceNew
   class HubspotDownloader
```

Comment:
I'm not actualy sure why you moved the hubspot downloader to a XXX_new  folder, was this for test purposes?  Anyway, as earlier, refactor all this to rename the folder  external_service  and not external_service_new